src/http.ts

```diff
@@ -1163,21 +1240,25 @@ export class LemmyHttp {
     endpoint: string,
     form: BodyType
   ): Promise<ResponseType> {
-    if (type_ == HttpType.Get) {
-      let getUrl = `${this.buildFullUrl(endpoint)}?${encodeGetParams(form)}`;
-      return fetch(getUrl, {
-        method: "GET",
+    if (type_ === HttpType.Get) {
+      const getUrl = `${this.buildFullUrl(endpoint)}?${encodeGetParams(form)}`;
+      const response = await fetch(getUrl, {
+        method: HttpType.Get,
         headers: this.headers,
-      }).then(d => d.json() as Promise<ResponseType>);
+      });
+
+      return await response.json();
```

Comment:
I wonder how it's letting you get away without specifying the return type here. I do like using await better than `then`, but you'll probably still have to do `as Promise<ResponseType>`

Reply:
`response.json()` returns a `Promise<any>`, however `wrapper` is already marked to return a `Promise<ResponseType>`. It's not showing an error because `any` can't really conflict with any types (it is `any` after all). So the manual cast isn't necessary for the function to return the expected type, but I can add it if you'd prefer it to be there.

Reply:
Gotcha, its fine then. I'd thought you couldn't coerce `any` to specific types, only the other way around.